### PR TITLE
Fix/billing and general improvements

### PR DESCRIPTION
**Pull request by tox2ik**
proofread, reword billing and other changes

### CODE REVIEW

File: docs/billing.md [hunk 1]
```diff
@@ -10,139 +10,171 @@ stack:         all
...
-You can test fortrabbit before you pay a dime. All you need to do is: sign up to fortrabbit and try it for free. The trial will end automatically, you don't have to cancel it. No credit card required. Once you decide to book a paid plan the following applies:
+You can test fortrabbit before paying a single a dime, completely for free. All you need to do is sign up and upload some code. The trial will end automatically, you don't have to cancel it and you will not be charged. No credit card required. Once you decide to book a paid plan, the following sections will apply.
 
 
 ### Consumption based billing
 
-Pay after usage: No upfront costs nor payment! **First you book, then you'll use it, then we'll invoice you** for what you have actually used. It's prorate, just like with your old telephone bill. At the end of the month you'll get an invoice for what you have used in that month. 
+There are no upfront costs or payments. The service is priced by usage, similar to how other cloud providers do it. This may break your expectations, but in practice this is how we do things;
 
+1. Customer signs up and opts in to pay for the service
```
Comment (frank-laemmer) [inline docs/billing.md]: Please use client instead of customer in general ( see [here](https://github.com/fortrabbit/help/#other-words ) ) , thanks. First line starts with "Customer", next line starts with "The customer". Suggestion: Maybe rewrite from third person to second person? "You" instead of "him" (client?)

File: docs/billing.md [hunk 2]
```diff
@@ -10,139 +10,171 @@ stack:         all
...
+2. The customer is using the platform for a certain amount of days
+3. At the end of the month, they receive an invoice for the preceding days
 
-### Daily billing cycle
+German users may recognize this as prorate, just like with the old telephone bill -
```
Comment (frank-laemmer) [inline docs/billing.md]: Why German users? "Prorated billing" is a general concept. Suggestion: Let's remove paragraph and use the prorate wording elsewhere above.

File: docs/billing.md [hunk 3]
```diff
@@ -10,139 +10,171 @@ stack:         all
...
 
 ### Payment methods
 
-You can pay by **credit card** (Visa, Master, Amex, no debit cards) or **SEPA direct debit** (EU clients only). For "enterprise clients" (large volume) we also offer to pay by **bill** (on account, after invoice).
+You can pay by credit card or SEPA direct debit. For enterprise clients (large volume) we also offer to pay by bill (giro / wire) (on account, after invoice).
+
+* We support credit cards including Visa, MasterCart and Amex.
+* SEPA direct debit is accessible to clients in the European Union.
+* We do not support debit cards.
+* We do not support payments via PayPal at this time and there are no future plans to integrate with PayPal.
```
Comment (frank-laemmer) [inline docs/billing.md]: Suggestion: Use less "We". 

* Credit cards … are supported
* Debit cards are not ...

File: docs/billing.md [hunk 4]
```diff
@@ -10,139 +10,171 @@ stack:         all
...
+* On the 3rd of the month, the fortrabbit billing bot will charge all invoices having credit card as the payment method.
+* Around the 7th of the month, all the SEPA direct payments invoices are issued and executed a few days later.
+
+In the middle of a month, a human will look over bounced invoices. At that time, due payments will be re-charged and additional payment reminders may be sent out.
 
 
 ### Costs monitoring
 
-You can check the current costs with your Billing Contact, see [here](/billing-contact#toc-costs-monitoring).
+You can check the current costs by looking at your *Billing Contact*, see link:/billing-contact#toc-costs-monitoring[here].
```
Comment (frank-laemmer) [inline docs/billing.md]: `link://` is that valid Markdown?

File: docs/billing.md [hunk 5]
```diff
@@ -10,139 +10,171 @@ stack:         all
...
+*Option 2*: Use the "Download / print" button from the link:/dashboard[Dashboard]: Login to the Dashboard &gt; Your Account &gt; Companies (Your Company) &gt; Billing Contact &gt; Invoice archive &gt; Invoice &gt; Download &amp; print invoice.
 
-Invoices are shown as HTML > you can use print from the browser to generate and download a nicely styled PDF.
+Invoices are shown as HTML. Depending on support in your browser, a PDF can be generated using the "Print to File" feature.
 
 
 ### Bounced payments
 
-Sometimes payments bounce, that just happens. Please mind that banks collect fees for those non-sufficient funds. We might pass those fees to you. There are various reasons why payments bounce: 
+Sometimes payments may bounce, that just happens. Regrettably, some banks may collect fees for failed charges or insufficient funds. We may pass those fees to you.
```
Comment (frank-laemmer) [inline docs/billing.md]: The fees are actually happening with our bank, so "some banks" does not apply.

File: docs/billing.md [hunk 6]
```diff
@@ -10,139 +10,171 @@ stack:         all
...
+### Pending transactions
 
-When you create a Billing Contact, we'll do a debit test on your credit card, when you enter the informations with us. This should not show up on your credit card statements, as it just a test. We have heard from clients that it sometimes shows up. Please don't worry, it will not be processed.
+When you create a Billing Contact, we'll do a debit test on your credit card. This should not show up on your credit card statements, because we will not finalize this trsansaction. Some clients have reported that this transaction shows up in their online banking. In any case, this reserved amount will not be claimed by us.
 
 
 ### Downloading previous invoices
 
-**Option 1**: Scan your e-mail inbox for mails with the subject "fortrabbit invoice" for the monthly invoices we have sent you. Each e-mail contains a secret link to the invoice. From there you can print/download.
+*Option 1*: Search your e-mail inbox for "fortrabbit invoice". Each invoice e-mail contains a secret link to the invoice. From there you can print or download the invoice.
```
Comment (frank-laemmer) [inline docs/billing.md]: Typographic detail: Please don't use italics if possible. https://github.com/fortrabbit/help#writing

**Comment by frank-laemmer** [comment]
It's hard to follow what's going here with green and red blocks for me. For example, the section about the measures on bounced invoices, the block have moved, so it's hard to compare.

I see that dedicated lines might help, but I prefer seeing things rendered in context. The rendering make a big difference to me. I can see if the paragraphs have a good length and if headlines are matching and how the TOC looks like.

So I am tempted to just merge and then continue editing. Also, it might help me to just see the new text. There is not so much need to compare against the old for me.

**Comment by frank-laemmer** [comment]
Regarding the conflicts: Sorry my bad I guess. I have changed the "Further reading" already, but in a different branch. So I think you use either changes to resolve this. With my changes I have also adjusted the spacing of the headlines:

```
text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text 


headline

text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text 
```

* two lines before headline
* one line after headline

File: docs/billing.md (hunk 2 above)
Comment (tox2ik) [reply docs/billing.md]: So then why have I never heard about "prorated billing" in my 32 years on this planet?

File: docs/billing.md (hunk 4 above)
Comment (tox2ik) [reply docs/billing.md]: I did not invent link:// it's used other places in this document.

File: docs/billing.md (hunk 1 above)
Comment (tox2ik) [reply docs/billing.md]: Client ok. I consider 2nd person voice to be less than ideal for technical documentation, also the list is connected to the paragraph above. I prefer the neutral tone.

File: docs/billing.md (hunk 2 above)
Comment (tox2ik) [reply docs/billing.md]: German users because presumably because you are talking about German telephone bill.

File: docs/billing.md (hunk 1 above)
Comment (tox2ik) [reply docs/billing.md]: You need to make a decision. I do not consider "maybe rewrite" as an imperative statement.

File: docs/billing.md (hunk 3 above)
Comment (tox2ik) [reply docs/billing.md]: ok.

File: docs/billing.md (hunk 5 above)
Comment (tox2ik) [reply docs/billing.md]: ok.

File: docs/billing.md (hunk 6 above)
Comment (tox2ik) [reply docs/billing.md]: did not mean to change that.

**Comment by tox2ik** [comment]
> * two lines before headline
> * one line after headline

I was aware of this rule, did not mean to break it.

File: docs/billing.md (hunk 1 above)
Comment (frank-laemmer) [reply docs/billing.md]: Your English is way better than mine + you are smarter than I am. So I suggest with a maybe, because maybe there is something I am missing.

File: docs/billing.md (hunk 2 above)
Comment (frank-laemmer) [reply docs/billing.md]: Let's remove the whole paragraph. I think it's not super helpful. The example is not good.

**Comment by frank-laemmer** [comment]
I have settled with "Further reading" not "Further Reading" now, since we use title case in headline.

**Comment by frank-laemmer** [comment]
@tox2ik I am a bit confused about the current state. It says outdated, clicking on changes ends up in a 404.

**Comment by frank-laemmer** [comment]
I have created a new proposal including my feedback with the last batch of commits. Please review or maybe merge right away. I'd say this collab is interesting but slow.

File: docs/billing.md (hunk 2 above)
Comment (tox2ik) [reply docs/billing.md]: fine by me.

File: docs/billing.md (hunk 1 above)
Comment (tox2ik) [reply docs/billing.md]: I changed this so it would fit in with the rest of the document.

File: docs/billing.md (hunk 4 above)
Comment (tox2ik) [reply docs/billing.md]: My bad. It was actually PHP-Storm that messed me up

File: docs/billing.md (hunk 6 above)
Comment (tox2ik) [reply docs/billing.md]: PHP-storm did some auto-magic (I messed up)
